src/components/Menu.tsx

```diff
@@ -97,8 +98,9 @@ const Menu = (): JSX.Element => {
       expand="md"
     >
       <div className="container p-0">
-        <Link to="/" className="navbar-brand" href="#">
-          CFTracker
+        <Link to="/" className="navbar-brand mt-2" href="#">
+        <img src={siteLogo} alt="haule" width={30} height={25}className="me-2 mb-2" />
+        CFTracker
```

Comment:
```suggestion
        <Link to="/" className="navbar-brand mt-2" href="#">
          <img src={siteLogo} alt="logo" width={30} height={25} className="me-2 mb-2" />
          CFTracker
```
Everything else looks great.

Reply:
🥲😅sorry was testing, done now 😃

Reply:
done, .. code formatting wasn't right ? i used prettify formatter , not sure if that outworked ...